Tools/ros2/ardupilot_msgs/srv/RallySet.srv

```diff
@@ -0,0 +1,13 @@
+
+# This service appends a Rally Point to the list or clears the whole list.
+
+# Rally Point.
+ardupilot_msgs/Rally rally
+
+# True to clear the whole list (rally entry is ignored).
+bool clear
+---
+# True if the action is successful.
+uint8 success
```

Comment:
Why is this not a bool if you call it `True` above?

Reply:
good catch, it is supposed to be a bool.